### PR TITLE
CODEOWNERS: Add maintainers for asps top-level directory

### PR DESCRIPTION
**Pull request by klendathu2k**
StarVMC/xgeometry defines geometry tags, and needs to be manually kept in sync with changes in StarVMC/Geometry.  Ownership should be the same for both areas.

asps/ contains the starsim core codes (as well as the root4star main application), so I need to be notified.  Added Victor as well.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -144,8 +144,9 @@
 /StarVMC/StarBASE                                  @klendathu2k
 /StarVMC/StarGeometry                              @klendathu2k
```
Comment (veprbl) [inline .github/CODEOWNERS:145]:
```suggestion
/StarVMC/StarGeometry                              @klendathu2k       @yfisyak
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -144,8 +144,9 @@
...
 /StarVMC/StarVMCApplication                        @yfisyak
-/StarVMC/xgeometry                                 @klendathu2k       @yfisyak
+/StarVMC/xgeometry                                 @klendathu2k
```
Comment (veprbl) [inline .github/CODEOWNERS]:
```suggestion
/StarVMC/xgeometry                                 @klendathu2k       @yfisyak
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (klendathu2k) [reply .github/CODEOWNERS:145]: No.  Software coordinators do not own the full STAR geometry.  (On request will add them to ownership of their individual subsystems).

File: .github/CODEOWNERS (hunk 2 above)
Comment (klendathu2k) [reply .github/CODEOWNERS]: No. Software coordinators do not own the full STAR geometry. (On request will add them to ownership of their individual subsystems).

**Review by plexoos — APPROVED** [review]
LGTM

**Pull request merged**